kpi/mixins/asset.py

```diff
@@ -0,0 +1,33 @@
+from collections import defaultdict
+from django.db.models import Prefetch
+
+from kobo.apps.organizations.models import Organization
+from kpi.models.asset import Asset
+
+
+class AssetViewSetListMixin:
+
+    def get_organizations_per_asset(self, asset_ids: list) -> dict:
```

Comment:
I'd prefer having a slightly more explicit function name like get_organizations_per_asset_id, because this implies you'll pass asset objects. Though it's not really important. What do you think?